spec/controllers/sms_broker/incoming_messages_controller_spec.rb

```diff
@@ -56,9 +56,9 @@ module SmsBroker
 
       it 'ensures correct encoding on incoming text' do
         pending('this test is broken')
```

Comment:
Not sure either, since it doesn't allow passing ISO-8851-1 at all anymore. Just delete it for now, perhaps it's not even a real problem. :/